.github/workflows/go.yml

```diff
@@ -45,13 +45,33 @@ jobs:
         make manifests
         git diff --exit-code
 
+    - name: Unit and Integration Tests
+      run: make test
+
     - name: Build image
       run: |
         docker build . -t ${built_image}
 
     - name: Create K8s Kind Cluster
       run: |
-        kind create cluster --image kindest/node:v1.19.7
+        cat <<EOF | kind create cluster --image kindest/node:v1.21.1 --config=-
+        kind: Cluster
+        apiVersion: kind.x-k8s.io/v1alpha4
+        nodes:
+        - role: control-plane
+          kubeadmConfigPatches:
+          - |
+            kind: ClusterConfiguration
+            apiServer:
+                extraArgs:
+                  v: "5"
+            controllerManager:
+                extraArgs:
+                  v: "5"
+            scheduler:
+                extraArgs:
+                  v: "5"
```

Comment:
Will this not cause a log flood on every build?
(it's 5 out of 10, so I guess it should not be very bad)

Reply:
5 out of 10 seems readable and detailed enough, the logs don't turn out a big mess

Reply:
maybe we can have it in a script like the kind-cluster-with-registry.sh script so the workflow will be more clear

Reply:
done